spell/psm-checklists.md

```diff
@@ -0,0 +1,113 @@
+# PSM Checklists
+
+## PSM Migration Checklist
+
+- [ ] IF contracts are being on-boarded in the current spell, execute the full [LitePSM On-boarding Checklist](#litepsm-on-boarding-checklist)
+- [ ] IF a PSM is deprecated in the migration, execute the full [PSM Off-boarding Checklist](#psm-off-boarding-checklist)
+- [ ] Amount of `gem` being migrated matches the Exec Sheet
+- [ ] Collateral amount that is fetched from the source PSM matches the collateral is added to the destination PSM
+- [ ] No bad debt is left behind
+- Source PSM:
+    - [ ] ilk matches source PSM
+    - [ ] IF present, `tin` matches the value in the Exec Sheet
+    - [ ] IF present, `tout` matches the value in the Exec Sheet
+    - [ ] `AutoLine` is updated according to the Exec Sheet
+- Destination PSM:
+    - [ ] ilk matches destination PSM
+    - [ ] IF present, `tin` matches the value in the Exec Sheet
+    - [ ] IF present, `tout` matches the value in the Exec Sheet
+    - [ ] `AutoLine` is updated according to the Exec Sheet
+    - [ ] `buf` matches the value in the Exec Sheet
+    - [ ] `fill` is called to ensure liquidity is immediately available after spell execution
```

Comment:
In all 3 phases and in the doc examples it is called conditionally (inside `if(rush() > 0)`). I think it's important to mention it here then